src/napari_deeplabcut/_widgets.py

```diff
@@ -22,6 +22,9 @@
 from napari.layers.utils.layer_utils import _features_to_properties
 from napari.utils.events import Event
 from napari.utils.history import get_save_history, update_save_history
+from PyQt5.QtWidgets import QLabel, QVBoxLayout, QDialog
+from PyQt5.QtSvg import QSvgWidget
+from PyQt5.QtGui import QPixmap
```

Comment:
Please use qtpy like the imports underneath, rather than a specific pyqt version.